pkg/ebpf/tracee.go

```diff
@@ -667,18 +675,18 @@ func (t *Tracee) populateBPFMaps() error {
 		if e.submit {
 			index := id / 8
 			offset := id % 8
-			configVal[16+index] = configVal[16+index] | (1 << offset)
+			configVal[80+index] = configVal[80+index] | (1 << offset)
 		}
 	}
 	if err = bpfConfigMap.Update(unsafe.Pointer(&cZero), unsafe.Pointer(&configVal[0])); err != nil {
 		return err
 	}
 
 	errmap := make(map[string]error, 0)
-	errmap["uid_filter"] = t.config.Filter.UIDFilter.Set(t.bpfModule, "uid_filter", uidLess)
-	errmap["pid_filter"] = t.config.Filter.PIDFilter.Set(t.bpfModule, "pid_filter", pidLess)
-	errmap["mnt_ns_filter"] = t.config.Filter.MntNSFilter.Set(t.bpfModule, "mnt_ns_filter", mntNsLess)
-	errmap["pid_ns_filter"] = t.config.Filter.PidNSFilter.Set(t.bpfModule, "pid_ns_filter", pidNsLess)
+	errmap["uid_filter"] = t.config.Filter.UIDFilter.Set(t.bpfModule, "uid_filter")
```

Comment:
Unrelated to your PR, but we should start the filterMap name once `t.config.Filter.UIDFilter` is created (so we don't have to give the 2nd argument whenever calling it).